lib/betterRules/compile.ts

```
@@ -0,0 +1,38 @@
+import { kebabCase } from 'lodash';
+import { Linter } from 'eslint';
+import { BetterRulesEntry, BetterRulesRecord } from './BetterRulesRecord';
+
+const compileRuleName = (ruleName: string): string => {
+  if (ruleName.includes('/')) {
```

What do you think about moving the `fixReactEs6Rule` functionality here? It would make the `defaultConfiguration` a bit cleaner and hide the weirdness of formatting the rule names. Or could that conflict with the names of other rules?

---

It was there initially.

However - I kind of want to treat the `betterRules` Folder as if it was a generic plugin.
And this is a special 'bug' for a special package (`react`) that we are using.

So I found the right place was actually 'outside'.

I was thinking about providing an option to compile to be able to `postProcess` rules and then put it there. But since, at least right now, it is only this one single rule, that would be a bit premature imho.

Long story short:
For now, I'd preffer to leave it outside of the `betterRules` package. Understandable?

---

Understandable :+1: